src/Components/styling/articleHomePage.css

```diff
@@ -0,0 +1,20 @@
+.article {
+    display:flex;
+    flex-wrap: wrap;
+}
+
+.articleCard{
+    align-items: center; /* Use another value to see the result */
+    width: 251px;
+    height: 345px;
```

Comment:
Using a set px value makes this unresponsive to different window sizes - try using a % as the unit here instead - that'll make the width responsive to the size of the parent element which in effect will make it scale with the page size.

This will make sure that the page scales and looks decent on both mobile and desktop layouts!